### DISCUSSION TITLE
Domain Recognition Issue

### DISCUSSION THREAD

**Discussion by AdvancedTherapies** [vercel/community#4884] [Help]
### Question

Hello,

I transferred a website from the team's account that developed it to my own account. However, Vercel doesn't recognize the domain and consistently indicates that another account is using it. I have followed the instructions to add a TXT verification record to the domain, but the issue persists.

The domain in question is: advancedtherapies.cibb.uc.pt

Could I please receive assistance in resolving this issue?

Thank you in advance!

**Comment by amyegan** [comment]
It doesn't look like the TXT record is coming through for `_vercel.uc.pt`. Can you double check that the record was added?

https://www.whatsmydns.net/#TXT/_vercel.uc.pt

**Reply by AdvancedTherapies** [reply]
It was added to `_vercel.cibb.uc.pt,` because the subdomain manager currently has permissions only at the `cibb.uc.pt` level, not at the `uc.pt` level

**Reply by amyegan** [reply]
The TXT record needs to be on `_vercel.uc.pt` for domain verification to work

**Reply by AdvancedTherapies** [reply]
To enable the subdomain manager to perform this change, a highly bureaucratic process is necessary due to the university domain's regulations.

If we can't find another way to solve this, I might have to switch platform as it is easier to change the A records on `advancedtherapies.cibb.uc.pt`  than adding TXT records on `_vercel.uc.pt`

Having the TXT record in `_vercel.cibb.uc.pt` does not prove that we manage it?
[https://www.whatsmydns.net/#TXT/_vercel.cibb.uc.pt](https://www.whatsmydns.net/#TXT/_vercel.cibb.uc.pt)

**Reply by amyegan** [reply]
The verification process needs to be done that way so our internal systems can verify ownership and prevent hijacking of subdomains. If you do not have access to update the DNS record, please contact the owner to add the TXT record for you. 🙏

**Reply by AdvancedTherapies** [reply]
I understand that, and really appreciate your help and compression, but there isn't other way we can verify the subdomain? A manual process?

As I said before, the process of adding  a TXT record on `_vercel.uc.pt` it's not feasible, I just will have to try to move to other platform and just add new A records 😕

**Reply by amyegan** [reply]
I double checked with the team. For domain security, we do not have a way to verify the subdomain without having the TXT record on `_vercel.uc.pt`.  If you can work with your domain admin to temporarily add the TXT record for verification purposes, that would let you add the subdomain to your project.
